### DISCUSSION TITLE
Support for 256 colors (24bit / truecolor)?

### DISCUSSION THREAD

**Discussion by TigerC10** [eclipse-theia/theia#12890] [Improvements]
Opening TheiaBlueprint on my MacOS, I've noticed some odd colors.  It seems Theia launches terminals with only 8 colors?
```shell
$> print $terminfo[colors]
8

$> echo $COLORTERM

```

Is there some hidden configuration setting to launch terminals with support for more colors?

**Comment by msujew** [comment]
All Theia does is spawn an instance of the terminal binary configured in your terminal profile. Does it work better in vscode for you? VSCode might use some better default arguments that we could copy.

**Reply by TigerC10** [reply]
Yes, VSCode is as I expect:
```shell
$> print $terminfo[colors]
256

$> echo $COLORTERM
truecolor
```

**Reply by msujew** [reply]
I see. Please create a feature request issue that outlines this difference between Theia and vscode.

**Reply by TigerC10** [reply]
Done #13523
